data/pigui/modules/time-window.lua

```diff
@@ -42,10 +42,14 @@ local function displayTimeWindow()
 		if requested == name and current ~= name then
 			color = colors.white
 		end
-		local time = name
 		-- translate only paused, the rest can stay
-		if time == "paused" then
+		local time = name
+		if (name == "paused") then
 			time = lc.PAUSED
+			local mpos = ui.getMousePos()
+			if mpos.x < pButt.x and mpos.y > pButt.y and ui.isMouseDoubleClicked(0) then
```

Comment:
Please use `ui.isItemHovered()` and move this test to after the `ui.coloredSelectedIconButton(...)` line (line 59).

Reply:
isItemHovered and the related family of isItem* functions operate on the interaction box of the last submitted item, which means they have to be run after the button is actually drawn; however, they remove the need to manually calculate and track the button interaction rect.

Reply:
tried without matching x, all buttons triggered

Reply:
That's because this function is run for each button - if you want the code to only run for the pause button, you need to replace the term that's calculating the rect with a term that checks for `time == paused`.